Mathlib/Algebra/Group/InjSurj.lean

```diff
@@ -51,6 +51,16 @@ namespace Injective
 
 variable {M₁ : Type*} {M₂ : Type*} [Mul M₁]
 
+/-- A type endowed with `*` has cancellative multiplication, if it admits an injective map that
```

Comment:
Could this be added for `IsCancelLeftMul` and `IsCancelRightMul`too?

Reply:
Sure! Here you go.